int32 message_code
bool successed
int32 robot_id
float64 robot_cost
int32[] performed_tasks
float64[] detailed_costs
int32 tasks_number

